core/src/main/scala/stryker4s/mutants/findmutants/MutantMatcher.scala

```diff
@@ -17,7 +17,7 @@ class MutantMatcher {
       matchStringMutators() orElse
       matchMethodMutators()
 
-  def matchBinaryOperators(): PartialFunction[Tree, Seq[Mutant]] = {
+  def matchBinaryOperators(implicit mutatorName: String = "BinaryOperator"): PartialFunction[Tree, Seq[Mutant]] = {
```

Comment:
Can't you add the variable in `MutationType`, implement it in each category, and then call it from the term (as they have an implicit conversion to `MutationType`)? Then you could also remove the implicit parameter

Reply:
I can't find a way to do this. Since we actually work with the mutation types.

Reply:
I gave it a try and pushed my changes in the `Mutant-name-in-MutationType` branch. Have a look, and if you agree with my changes then you can merge it. The name now comes from whatever it is mutating to, instead of what it is mutating from (which should be the same in every case)

Reply:
I totally didn't think of that solution i will take a look into that tomorrow + testing